### PR TITLE
add a monorepo config for @wordpress packages

### PR DESCRIPTION
**Pull request by blowery**
The `@wordpress/*` packages come from a monorepo and we would like to group renovate PRs for them. This updates the config to add a grouping for those packages.

See https://renovatebot.com/docs/configuration-options/#packagerules for config settings.

### CODE REVIEW

**Comment by blowery** [comment]
@rarkins Does this look like the right way to configure a monorepo?

File: renovate.json [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-  ]
+	"extends": [ "config:base", "default:pinDigestsDisabled" ],
+	"statusCheckVerify": true,
+	"ignoreDeps": [ "jquery" ],
+	"labels": [ "Framework", "[Type] Task", "[Status] Needs Review" ],
+	"packageRules": [
+		{
+			"packagePatterns": "^@wordpress/",
+			"groupName": "WordPress",
+			"groupSlug": "wordpress",
```
Comment (rarkins) [inline renovate.json]: There's no need to manually configure this as it will be calculated from the `groupName`

**Comment by rarkins** [comment]
@blowery better yet, how about we add this as a default/preset grouping so anyone using Renovate's `config:base` preset gets it automatically?

**Comment by rarkins** [comment]
I now created a preset `group:wordpress` that does this and is included automatically with `config:base`

**Comment by blowery** [comment]
Thanks @rarkins!

**Pull request closed without merging**